src/scenes/AcceleratorRouter/Documents/DocumentView/index.tsx

```diff
@@ -31,13 +31,20 @@ export default function DocumentView(): JSX.Element {
   const [showUpgradeModal, setShowUpgradeModal] = useState(false);
   const [outlinePanelOpen, setOutlinePanelOpen] = useState(true);
 
-  const latestManifestId = useMemo(() => documentTemplate?.variableManifestId ?? -1, [documentTemplate]);
```

Comment:
These sorts of usages of `useMemo` are actually performance robbing. If we are just calculating a property of an object, we should just set it as a regular variable, and it will be recomputed on every component refresh. This is OK because it costs less than what the `useMemo` needs to determine if the value needs to be recomputed or not.

Reply:
Great point! 💯